### PR TITLE
Resolving First timers issue template grammatical mistakes

### PR DESCRIPTION
**Pull request by ojninja16**
Fixes #11395  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by ojninja16** [comment]
@TildaDares Please review....

File: .github/first-timers-issue-template.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 
+Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
 
-If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open source software!
```
Comment (TildaDares) [inline .github/first-timers-issue-template.md:3]:
```suggestion
If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!
```

File: .github/ISSUE_TEMPLATE/--first-timers-only.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 
-****
+---
 
 Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
 
-If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!
+If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open source software!
```
Comment (TildaDares) [inline .github/ISSUE_TEMPLATE/--first-timers-only.md:17]:
```suggestion
If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Hi @ojninja16, feel free to just click the `Commit suggestion` button. Thanks!

**Review by TildaDares — APPROVED** [review]
Amazing work @ojninja16. Thank you!!

**Comment by ojninja16** [comment]
Glad, I could help.....thanks for the opportunity @TildaDares

**Pull request merged**